mlflow/tracking/fluent.py

```diff
@@ -1392,6 +1397,7 @@ def setup_autologging(module):
             ] = True
             if not autologging_params.get("disable", False):
                 _logger.info("Autologging successfully enabled for %s.", module.__name__)
+            check_and_log_warning_for_unsupported_integration(autolog_fn.integration_name)
```

Comment:
I don't think we need this logic since we're calling `autolog_fn()` immediately above, which should log the same warning message.

For example, the following workflow produces two identical warning messages:

```
import mlflow
mlflow.autolog()

from sklearn import datasets
from sklearn.linear_model import LogisticRegression

iris = datasets.load_iris()

logistic = LogisticRegression(solver="saga", max_iter=200, random_state=0)
logistic.fit(iris.data, iris.target)
```

```
2021/03/01 23:27:01 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of sklearn. If you encounter errors during autologging, try upgrading / downgrading sklearn to a supported version, or try upgrading MLflow.
2021/03/01 23:27:02 INFO mlflow.tracking.fluent: Autologging successfully enabled for sklearn.
2021/03/01 23:27:02 WARNING mlflow.utils.autologging_utils: You are using an unsupported version of sklearn. If you encounter errors during autologging, try upgrading / downgrading sklearn to a supported version, or try upgrading MLflow.
2021/03/01 23:27:02 INFO mlflow.utils.autologging_utils: Created MLflow autologging run with ID '6baa8a55e05d47d2be5bc5c234edb27b', which will track hyperparameters, performance metrics, model artifacts, and lineage information for the current sklearn workflow
```